src/ZendeskApi.Build/ZendeskApi.Commons.props

```diff
@@ -0,0 +1,14 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
```

Comment:
We should also add a description to both projects - its currently auto-populated.